Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM ubuntu:14.04
+MAINTAINER Jussi Vaihia <jussi.vaihia@futurice.com>
+
+RUN groupadd app && useradd --create-home --home-dir /home/app -g app app
```

Comment:
`useradd -m app` is a bit shorter and is equivalent to what you do. No need for separate groupadd command. Also I'd move the user creation lower in the Dockerfile, after the "heavy" stuff.